shell/plugins/steve/subscribe.js

```diff
@@ -601,9 +597,9 @@ const defaultActions = {
   },
 
   rehydrateSubscribe({ state, dispatch }) {
-    if ( process.client && state.wantSocket && !state.socket ) {
-      dispatch('subscribe');
-    }
+    // if ( state.wantSocket && !state.socket ) {
```

Comment:
Looks like this needs to come back

Reply:
you are correct @codyrancher . Thanks for spotting this 🙏